python/hail/dataset.py

```diff
@@ -1433,53 +1471,51 @@ def export_vcf(self, output, append_to_header=None, export_pp=False, parallel=Fa
 
         >>> hc.read('data/example.vds').export_vcf('data.example.vcf.bgz', export_pp=True, parallel=True)
 
-        Export a sites-only VCF with addition header lines from `myheader.txt`:
+        Export a sites-only VCF with additional header lines from *myheader.txt*:
 
         >>> (hc.read('data/example.vds')
         >>>    .filter_samples_all()
         >>>    .export_vcf('data/sites-only.vcf.bgz', append_to_header='data/myheader.txt')
 
         **Notes**
-        py:meth:`exportvcf` exports the VDS the VCF format as described in the [VCF 4.2 spec](https://samtools.github.io/hts-specs/VCFv4.2.pdf).
+        py:meth:`exportvcf` writes the VDS to disk in VCF format as described in the `VCF 4.2 spec <https://samtools.github.io/hts-specs/VCFv4.2.pdf>`_.
```

Comment:
I just checked and both links look and work fine in the built docs.  Not sure what you're seeing.